docs/svelte-testing-library/api.mdx

```diff
@@ -30,6 +31,32 @@ import {render} from '@testing-library/svelte'
 const {results} = render(YourComponent, {ComponentOptions}, {RenderOptions})
 ```
 
+## `screen`
```

Comment:
To keep the docs consistent we should also revert this change because no other libraries include it in their docs.
`screen` is not Svelte specific and is already documented in the core api, ideally we don't want to introduce duplicates.

Reply:
That makes sense, I reverted this.